src/App.css

```
@@ -1,7 +1,7 @@
 * {
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-  'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-  sans-serif !important;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```

Stylelint (17.12.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)